Content.Server/Shuttles/Systems/EmergencyShuttleSystem.cs

```diff
@@ -235,8 +236,18 @@ private void OnEmergencyFTLComplete(EntityUid uid, EmergencyShuttleComponent com
                 [ShuttleTimerMasks.ShuttleTime] = countdownTime,
                 [ShuttleTimerMasks.SourceTime] = countdownTime,
                 [ShuttleTimerMasks.DestTime] = countdownTime,
-                [ShuttleTimerMasks.Text] = new string?[] { "BYE!" }
             };
+
+            // by popular request
+            // https://discord.com/channels/310555209753690112/770682801607278632/1189989482234126356
+            if (new Random().Next(1000) == 0)
```

Comment:
Please use `IRobustRandom` to generate a random number. It's already a dependency of this system and called `_random`